web/app/css/breadcrumb-header.css

```diff
@@ -0,0 +1,17 @@
+@import 'styles.css';
+
+.ant-layout-header {
+  background-color: var(--header-black);
+  z-index: 100;
+  position: fixed;
+  width: 100vw;
+  padding: 22px 40px;
+
+  & span.ant-breadcrumb-link, & span.ant-breadcrumb-separator, & :first-child {
+     color: var(--white);
+  }
+}
+
+.ant-layout-header:last-child {
```

Comment:
I think it would be cool if breadcrumbs were super big, like `h1` sized, and bold. And then we could remove the page headers from each page!

Something like:
```
@import 'styles.css';

.ant-layout-header {
  background-color: #26e99d;
  height: 84px;
  z-index: 100;
  position: fixed;
  width: 100vw;
  padding-top: 20px;

  & span.ant-breadcrumb-link, & span.ant-breadcrumb-separator, & :first-child {
     color: black;
     font-size: 24px;
     font-weight: 700;
  }
}
```